be/src/runtime/user_function_cache.cpp

```diff
@@ -216,10 +216,12 @@ Status UserFunctionCache::_get_cache_entry(int64_t fid, const std::string& url,
     std::string file_name = _get_file_name_from_url(url);
     {
         std::lock_guard<std::mutex> l(_cache_lock);
-        auto it = _entry_map.find(fid);
-        if (it != _entry_map.end()) {
-            entry = it->second;
-        } else {
+        for (const auto& item : _entry_map) {
+            if (item.second->checksum == checksum) {
+                entry = item.second;
```

Comment:
add break here?

Reply:
Thanks for your suggestion, I changed it.